src/main/views/income-expense-scripts.njk

```diff
@@ -0,0 +1,5 @@
+<script src="{{ asset_paths['js'] }}/income-expense-calculation.js"></script>
+<script src="{{ asset_paths['js_vendor'] }}/nodelist-foreach-polyfill.js"></script>
+<script src="{{ asset_paths['js_vendor'] }}/classlist-polyfill.js"></script>
+<script src="{{ asset_paths['js_vendor'] }}/startswith-polyfill.js"></script>
```

Comment:
Out of interest, are classlist-polyfill.js and startswith-polyfill.js in use?

Reply:
not currently, i'm reworking this atm